docs/guides/developer/docs/release-manager.md

```diff
@@ -1,474 +1,366 @@
 
-### Creation of Release Branch
+### Release Branch
 
 According to the set release schedule, at one point a release branch should be cut, effectively marking a feature freeze
-for a given release.
-
-Once a release process has started, a branch should be created with the proposed version number of that release. This
-branch is created off of `develop` and should be named `r/N.x` (e.g. `r/3.x` for the Opencast 3.0 release branch).
-In the following command, we assume that the release branch for 3.0 is to be created. For other versions, please adjust
-the version number accordingly.
+for a given release.  This branch is split of `develop` and should be named `r/N.x` (e.g. `r/6.x` for the Opencast 6
+release branch).
 
-Create the Branch:
+Example on how to create the Opencast 7 release branch:
 
-1. Grab the merge ticket for `develop`. You are going to make changes there and it becomes ugly if someone else changes
-   something at the same time.
 
-2. Check out `develop` and make sure it has the latest state (replace `<remote>` with your remote name for the community
+1. Check out `develop` and make sure it has the latest state (replace `<remote>` with your remote name for the community
    repository):
 
         git checkout develop
         git pull <remote> develop
 
-3. Create and push release branch:
+2. Make sure you did not modify any files. If you did, stash those changes:
 
-        git checkout -b r/3.x
-        git push <remote> r/3.x
+        git status   # check for modified files
+        git stash    # stash them if necessary
 
-4. Make sure you did not modify any files. If you did, stash those changes.
+3. Create and push the new release branch:
 
-5. That is it for the release branch. Now lets make sure we update the versions in `develop` in preparation for the next
-   release:
+        git checkout -b r/7.x
+        git push <remote> r/7.x
+
+5. That is it for the release branch. Now update the versions in `develop` in preparation for the next release:
 
         git checkout develop
         for i in `find . -name pom.xml`; do \
-          sed -i 's/<version>3.0-SNAPSHOT</<version>4.0-SNAPSHOT</' $i; done
+          sed -i 's/<version>6-SNAPSHOT</<version>7-SNAPSHOT</' $i; done
 
-6. Have a look at the changes. Make sure no library version we use has the version `3.0-SNAPSHOT` and was accidentally
+6. Have a look at the changes. Make sure no library version we use has the version `6-SNAPSHOT` and was accidentally
    changed. Also make sure that nothing else was modified:
 
         git diff
+        git status | grep modified: | grep -v pom.xml   # this should have no output
 
-7. Check again for unwanted changes.
-
-8. If everything looks fine, commit everything and push it to the community repository:
+8. If everything looks fine, commit the changes and push it to the community repository:
 
-        git commit -as
+        git add $(git status | grep 'modified:.*pom.xml' | awk '{print $2;}')
+        git commit -s
```

Comment:
I personally have a strong aversion against the `NOJIRA` since it gives you no additional information at all. Others seem to like it though.

To not argue about this, I suggest we avoid the messages for now. Right now, the only place we have a suggestion is the release commit which does not use the `NOJIRA` and [in practice mostly hasn't as well](https://paste.fedoraproject.org/paste/XziB7-s9BBXbdUHNSjGgrQ).